ci-operator/step-registry/openstack/scale/masters/openstack-scale-masters-commands.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+if [ -z "$SCALE_MASTERS_FLAVOR_ALTERNATE" ]; then
+	info "All defined environment variables for the step are empty. Exiting."
+	exit 0
+fi
+
+declare patch=''
+
+function info() {
+	printf '%s: %s\n' "$(date --utc +%Y-%m-%dT%H:%M:%SZ)" "$*"
+}
+
+function wait_for_cpms_done() {
+  if
+    ! oc wait --timeout=90m --for=condition=Progressing=false controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.spec.replicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.status.updatedReplicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.status.replicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.status.readyReplicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null; then
+      info "CPMS not scaled to 3 replicas"
+      oc get controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster
+      oc describe controlplanemachineset.machine.openshift.io cluster --namespace openshift-machine-api
+      exit 1
+  else
+    info 'CPMS scaled the masters.'
+    oc get controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster
```

Comment:
ditto

Reply:
wrong?
See the logs:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41379/rehearse-41379-pull-ci-openshift-installer-master-e2e-openstack-ovn-masterscale-zones/1681831208356417536/artifacts/e2e-openstack-ovn-masterscale-zones/openstack-scale-masters/build-log.txt

We can see the oc output.

Reply:
duh, you're right. I don't know what I looked at.